tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -125,6 +130,53 @@ def test_build_image(
     _test_export_logs(s3_bucket_factory, image, region)
 
 
+@pytest.mark.usefixtures("instance")
+def test_kernel4_build_image_run_cluster(
+    region,
+    os,
+    pcluster_config_reader,
+    architecture,
+    images_factory,
+    request,
+    scheduler_commands_factory,
+    clusters_factory,
+    scheduler,
+):
+    """Test build image for given region and os and run a job in a new slurm cluster created from the new images"""
+
+    image_id = generate_stack_name("integ-tests-build-image", request.config.getoption("stackname_suffix"))
```

Comment:
minor I'd move variable initialization just after the usage

Reply:
Done